lib/northbound.c

```diff
@@ -1014,45 +1207,48 @@ static int nb_callback_configuration(const enum nb_event event,
 			break;
 		default:
 			flog_err(EC_LIB_DEVELOPMENT,
-				 "%s: unknown event (%u) [xpath %s]",
-				 __func__, event, xpath);
+				 "%s: unknown event (%u) [xpath %s]", __func__,
+				 event, xpath);
 			exit(1);
 		}
 
 		flog(priority, ref,
-		     "%s: error processing configuration change: error [%s] event [%s] operation [%s] xpath [%s]",
-		     __func__, nb_err_name(ret), nb_event_name(event),
+		     "error processing configuration change: error [%s] event [%s] operation [%s] xpath [%s]",
+		     nb_err_name(ret), nb_event_name(event),
 		     nb_operation_name(operation), xpath);
+		if (errmsg[0] != '\0')
```

Comment:
Sure, although `strlen` is a bit more obvious

Reply:
Indeed :)

I updated the code to use `strlen() > 0` instead.